cli/src/command/account.rs

```diff
@@ -627,10 +627,10 @@ pub async fn outputs_command(account: &Account) -> Result<(), Error> {
     if outputs.is_empty() {
         println_log_info!("No outputs found");
     } else {
-        let output_ids: Vec<OutputId> = outputs.iter().map(|o| o.output_id).collect();
-        println_log_info!("Outputs: {output_ids:#?}");
+        for (i, output_data) in outputs.into_iter().enumerate() {
```

Comment:
Do the same for the `unspent_outputs_command()`?

Reply:
Yeah right, would be nice

Reply:
fixed 
(also added the `Outputs` and `Unspent outputs:` which were there berfore